master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -156,3 +175,180 @@ func (a *agentResourceManager) aggregateTaskSummaries(
 	}
 	return summaries
 }
+
+func (a *agentResourceManager) getResourcePoolConfig(poolName string) (ResourcePoolConfig, error) {
+	for i := range a.poolsConfig.ResourcePools {
+		if a.poolsConfig.ResourcePools[i].PoolName == poolName {
+			return a.poolsConfig.ResourcePools[i], nil
+		}
+	}
+	return ResourcePoolConfig{}, errors.Errorf("cannot find resource pool %s", poolName)
+}
+
+func (a *agentResourceManager) createResourcePoolSummary(
+	ctx *actor.Context,
+	poolName string,
+) (*resourcepoolv1.ResourcePool, error) {
+	const na = "N/A"
```

Comment:
suggestion: leave out? it should make it for clients to have an easier time to tell this is missing/unset.

Reply:
Same comment as above. This serves to differentiate between an unset value and the situation where the value would be nonsensical. I'm not super attached to it and we can change it if you'd like, but it was an intentional choice.

Reply:
thanks for updating the other one. 
could we get rid of this N/A the same as here https://github.com/determined-ai/determined/pull/1724#discussion_r558471801

a response from master: 
```
{
  "resourcePools": [
    {
      "name": "default",
      "description": "",
      "type": "RESOURCE_POOL_TYPE_STATIC",
      "numAgents": 1,
      "slotsAvailable": 1,
      "slotsUsed": 0,
      "cpuContainerCapacity": 100,
      "cpuContainersRunning": 0,
      "defaultGpuPool": true,
      "defaultCpuPool": true,
      "preemptible": false,
      "minAgents": 0,
      "maxAgents": 0,
      "cpuContainerCapacityPerAgent": 100,
      "schedulerType": "SCHEDULER_TYPE_FAIR_SHARE",
      "schedulerFittingPolicy": "FITTING_POLICY_BEST",
      "location": "on-prem",
      "imageId": "N/A",
      "instanceType": "N/A",
      "masterUrl": "",
      "masterCertName": "",
      "startupScript": "",
      "containerStartupScript": "",
      "agentDockerNetwork": "",
      "agentDockerRuntime": "",
      "agentDockerImage": "",
      "agentFluentImage": "",
      "maxIdleAgentPeriod": 0,
      "maxAgentStartingPeriod": 0,
      "details": {
        "aws": null,
        "gcp": null,
        "priorityScheduler": null
      }
    }
  ],
  "pagination": {
    "offset": 0,
    "limit": 0,
    "startIndex": 0,
    "endIndex": 1,
    "total": 1
  }
}

```

on